mealy/error_visualizer.py

```diff
@@ -199,9 +201,11 @@ def plot_feature_distributions_on_leaves(self, leaf_selector='all_errors',
             show_class (bool): show the proportion of Wrongly and Correctly predicted samples in the feature
                 distributions.
 
-            rank_leaves_by (str): ranking criterium for the leaf nodes. It can be either 'purity' to rank by the leaf
-                node purity (ratio of wrongly predicted samples over the total for an error node) or 'class_difference'
-                (difference of number of wrongly and correctly predicted samples in a node).
+            rank_leaves_by (str): ranking criterion for the leaf nodes. Valid values are:
+                * 'global_error': rank by the global error (% total error in the node)
+                * 'purity': rank by the purity (ratio of wrongly predicted samples over the total for an error node)
```

Comment:
Same comment for error analyzer (get_ranked_leaf_ids)
```suggestion
                * 'purity': rank by the purity (ratio of wrongly predicted samples over the total number of node samples)
```
WDYT? Seems a bit clearer to me?

Reply:
yes, better, thx